lib/pbench/client/__init__.py

```diff
@@ -49,6 +49,7 @@ class API(Enum):
     DATASETS_VALUES = "datasets_values"
     ENDPOINTS = "endpoints"
     KEY = "key"
+    QUISBY = "quisby"
```

Comment:
This needs to be updated now.

It would be great to add a functional test case using the `visualize` API on the `uperf` run, and do some validation of the returned JSON payload. (Which also would have caught this oversight!)

Reply:
Fixed the client API enum. Sure @dbutenhof I'll add functional tests(both single dataset and compare dataset) in a separate PR